### PR TITLE
New release v2.5.0

### PR DESCRIPTION
**Pull request by holgerd77**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -5,6 +5,98 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
...
+### Special Thanks
+
+Special thanks to:
+
+- @mattdean-digicatapult for his indefatigable work on the new StateManager interface and for fixing a large portion of the failing blockchain tests
+- @rmeissner for the work on Constantinople 
+- @vpulim for jumping in so quickly and doing a reliable ``SSTORE`` implementation within 4 days
+- @s1na for the new API test suite
+
+Many others did valuable contributions which can't all be listed here, many thanks to all!
```
Comment (axic) [inline CHANGELOG.md]: Curious: why it cannot be listed?

One possible way is what Solidity is doing, just listing all the user handles.

Comment (holgerd77) [reply CHANGELOG.md]: You're right, will update.

**Comment by holgerd77** [comment]
Ah, @rmeissner, @mattdean-digicatapult, @vpulim, @axic, dismissed all your reviews by a force-push, if someone of you could re-approve I would be happy! 😄

**Comment by holgerd77** [comment]
@rmeissner Thanks, I was actually live-waiting in front of my laptop, hoping that someone of you would be awake / online / aware / free-of-duties! 😛 

Will now merge and then do the release.

**Pull request merged**